Dockerfile

```diff
@@ -28,8 +28,15 @@ ENV CT_VER="3.1.1"
 
 # install dependencies
 RUN apt-get update && \
-    apt-get install --no-install-recommends -y curl git sudo && \
+    apt-get install --no-install-recommends -y curl git sudo openssh-client && \
     apt-get clean && rm -rf /var/lib/apt/lists/*
+
```

Comment:
Minor issue: in perspective, we want to make `abs` a part of our "open source app platform" tooling, available for anyone willing to build and test apps for our platform. People in the community might use different git repos for storing catalogs then just github - and then it will fail for them. Or they might not use git for that at all.

To me it feels like the problem of where and how to store artifacts built (chart file, metadata files) should be out of scope of `abs` and - in our case - handled by CI/CD (so it checks out the repo, runs `abs`, then commits new files to app catalog's repo). 
WDYT?

Reply:
Yeah fine with me. I'll leave this in while I test this with architect-orb. After that the plan is to make uploading independent from packaging in CI